### DISCUSSION TITLE
Quitt (DQTS)

### DISCUSSION THREAD

**Discussion by luziusmeisser** [Frankencoin-ZCHF/Frankencoin#24] [Acceptable Collaterals]
I propose to add DQTS tokens to the list of supported collaterals. The service behind this company has been growing steadily for more than a decade and achieves a yearly revenue of roughly 4 million CHF. However, since this is a small company without liquid market, I set the reserve ratio at a very high 50%. So half of the minted ZCHF go directly into the reserve and the minters face an extremely high cost in case of a liquidation. At the same time, the interest is only 1%, but since the interest is also paid on the reserve, it is effectively a 2% interest rate on the usable amount, which is in line with current interest rates in Switzerland. 

https://www.frankencoin.com/position/0x7F0306368Ae8Df8718D6F79cffc53A38bF6E981C

**Comment by luziusmeisser** [comment]
After consulting with the other major shareholders, we suggest to revive DQTS as collateral with the following parameters:

<img width="1882" height="1566" alt="image" src="https://github.com/user-attachments/assets/1b4d6347-3f30-460d-803b-7c15da581c95" />

In the meantime, revenue has grown to 6.2 million CHF, or 5.5 million CHF for the Swiss entity, which is nice. You can find the annual accounts of ServiceHunter AG here:

<img width="884" height="1551" alt="image" src="https://github.com/user-attachments/assets/897b55ca-2da9-497d-8953-f5df59744f9e" />

<img width="1015" height="1557" alt="image" src="https://github.com/user-attachments/assets/d1acda4a-4bb8-4107-a25f-5459b724480f" />

The company offers the administration of household employments, helping private employers to fulfill all their legal obligations (social security, taxes, accident insurance, etc.) and charging a fee for that. Overall, the company has a strong and profitable core business in Switzerland. The growth was mostly achieved growth by investing into an international expansion. However, the subsidiaries in Germany and the UK are not profitable yet. Furthermore, quitt has rolled out business services in Switzerland to complement the private offering. If these initiatives succeed, there should be further robust (and profitable) growth for the foreseeable future. There is a substantial share transaction planned at 8 CHF per share, which corresponds to a valuation of 16 million, which seems fair for a SaaS company with 5.5 million in revenues but not much substance value.

More information can be found on the company's [investor relations page](https://shares.quitt.ch/), on which one can also buy and sell its share tokens. It currently has 229 shareholders, mostly coming from a crowd funding done in 2012.

**Comment by luziusmeisser** [comment]
In the light of issue https://github.com/Frankencoin-ZCHF/Frankencoin/issues/52, I think it makes sense to increase the auction duration to 72 hours. So in the worst case, the price would go from 2300 CHF to 0 CHF over the course of 259200 seconds, which is about 0.10 CHF per block. With a reserve ratio of 40%, this still leaves 24 blocks for a profitable liquidation at a starting price of 6 CHF, which if ok.

**Comment by samclassix** [comment]
Thanks for sharing the updated accounts. Putting together a few observations for the community's consideration — **not taking a position on the proposal itself. I do not claim that my observations are correct or sufficient to take a decision — DO YOUR OWN RESEARCH.**

**What we now know from the UK subsidiary (Paye For Nannies Ltd, year ended 31 Dec 2025)**

The UK accounts were filed at Companies House on 16 March 2026 and are now publicly available. Under the small companies regime, no income statement is filed — but the balance sheet tells enough.

Shareholders' funds fell from £221,752 to £181,043 — an implied net loss of **~£40,709** for 2025, the first full year under ServiceHunter ownership. The headline cash of £772,768 is misleading: £346,456 of it is social security and tax liabilities owed to HMRC (standard for a PAYE processing company), leaving effective free cash of ~£426k. The company sold its freehold property (NBV £377k) during the year and used the proceeds to pay off all bank debt. The UK entity is now debt-free — but only because its primary fixed asset was liquidated. Employees declined from 17 to 15. Accounts are unaudited.

**Group picture — three entities, one set of public accounts**

The community still does not have a consolidated view of the group:

| Entity                          | Shareholder equity   | Accounts available?     |
| ------------------------------- | -------------------- | ----------------------- |
| quitt AG (Swiss parent)         | CHF 1,638,531        | Yes (audited)           |
| Paye For Nannies Ltd (UK)       | £181,043 ≈ CHF ~207k | Yes (unaudited, no P&L) |
| quitt Deutschland GmbH (Munich) | Unknown              | No public filing found  |

The Swiss parent carries "Participations" (the two subsidiaries combined) at **CHF 1,425,499**. The UK subsidiary alone is worth ~CHF 207k on book — implying the participations line is carried at roughly 7× the UK book value, with the German entity's contribution unknown but likely negative given CHF 700k of subsidiary loans have already been impaired.

On a consolidated basis, group equity is almost certainly materially lower than the CHF 1.64M shown for the Swiss entity alone.

**Valuation — five methods based on available accounts**

- P/S (1.5–2.5× revenue, −30% illiquidity): CHF 2.9–4.8 / share
- EV/EBITDA (5–8×, −30% illiquidity): CHF 2.6–4.2 / share
- P/E (20× reported net profit of CHF 111k): CHF 1.1 / share — and ~CHF 0.08 on recurring profit only
- DCF (FCF ~CHF 1.1M, 14–18% WACC, −30% illiquidity): CHF 2.5–4.2 / share
- Book value (tangible, after intangibles haircut): CHF 0.10–0.47 / share

All five methods suggest fair value well below the proposed CHF 6 collateral price and the CHF 8 last traded price.

**Comparison with two listed Frankencoin positions — LENDS and BOSS**

Three equity token positions now exist in the protocol, each calibrated differently. The pattern is instructive.

| Metric                          | LENDS (lend.ch) | Boss Info AG (FY2025) | ServiceHunter (DQTS)       |
| ------------------------------- | --------------- | --------------------- | -------------------------- |
| Market cap                      | CHF 48.4M       | CHF 79.8M             | CHF 16.0M                  |
| Total shares                    | 31,066          | 10,650,100            | 2,000,000                  |
| Tokenized on-chain              | 27,277 (87.8%)  | 10,650,100 (100%)     | 1,134,480 (56.7%)          |
| On-chain shareholders           | 283             | 160                   | 229                        |
| Parent equity (token basis)     | Unknown         | **CHF 14,538,987**    | **CHF 1,638,531**          |
| Consolidated group equity       | Unknown         | CHF 362,014           | Unknown — no filing        |
| Group revenue                   | Unknown         | CHF 36.1M             | CHF 5.49M (Swiss only)     |
| Group operating cash flow       | Unknown         | CHF 5.74M             | Unknown                    |
| Frankencoin mint limit          | **CHF 1M**      | CHF 7M                | CHF 5M                     |
| Usable (net of 40% reserve)     | **CHF 600k**    | CHF 4.2M              | CHF 3.0M                   |
| **Usable / market cap**         | **1.2%**        | **5.3%**              | **18.8%**                  |
| **Usable vs. parent equity**    | Unknown         | **29%**               | **183%**                   |
| Tokens locked for full position | 53.9% tokenized | 8.8% of all tokens    | 73.5% of tokenized         |
| Free float after full position  | 46.1% tokenized | 91.2% of all tokens   | 15% of total shares        |
| Consolidated accounts filed     | Unknown         | Yes, audited          | No                         |
| Subsidiary losses / impairments | Unknown         | None disclosed        | CHF 700k impaired, ongoing |

### The community calibrated LENDS at 1.2% of market cap and BOSS at 5.3%. ServiceHunter is proposing **18.8%** — on the smallest, least profitable, and least transparent of the three companies.

### A simple question worth asking: LENDS has roughly 3× the market cap of DQTS and was approved for CHF 1M. Why is DQTS — with 3× worse fundamentals — requesting 5× that limit?

Boss Info's usable mint of CHF 4.2M is 29% of its CHF 14.54M parent equity, backed by CHF 36.1M group revenue and CHF 5.74M operating cash flow. The thin consolidated equity is a holding-company artifact; the operating substance is clearly there.

ServiceHunter's usable mint of CHF 3.0M **exceeds the entire Swiss parent equity of CHF 1.64M by 83%**. There is no consolidated filing, no disclosed group cash flow, a loss-making UK subsidiary in its first year, and a German subsidiary that has absorbed CHF 700k+ in impaired loans since 2022.

**Red flags from the accounts (unchanged)**

- EBIT of CHF 24k on CHF 5.49M revenue — 0.4% margin
- Recurring net profit stripped of one-off extraordinary income: ~CHF 8k
- D&A of CHF 1.47M nearly equals entire EBITDA of CHF 1.50M — driven by acquisition amortisation
- IT costs up 5.6× year-on-year (CHF 91k → CHF 509k)
- CHF 700k loan impairment already taken on subsidiary balances
- Book equity of CHF 1.64M vs CHF 16M stated market cap (9.8× price/book)
- No consolidated accounts — group losses remain undisclosed
- UK subsidiary: implied net loss of ~£40,709 in first year under new ownership
- UK subsidiary: primary fixed asset (freehold property) disposed; entity now asset-light

**Token structure and liquidation mechanics**

ServiceHunter has 2,000,000 total shares, but only **1,134,480 are tokenized on-chain (56.7%)**. The remaining 865,520 shares are non-tokenized — completely invisible to any Frankencoin liquidation auction. The protocol can only sell what exists on-chain.

At a collateral price of CHF 6 and a mint limit of CHF 5,000,000, a full position would lock **833,333 DQTS — 73.5% of the entire tokenized supply**. That leaves a free float of just 301,147 DQTS, representing 15% of total company shares.

For context: LENDS locks 53.9% of its tokenized supply for a CHF 1M position — accepted precisely because CHF 600k usable is small enough that an imperfect auction is manageable. BOSS locks 8.8% of total supply, leaving 91.2% free float across 160 holders. DQTS at 73.5% lock with CHF 3M at stake is the worst of both: a large absolute exposure and a structurally illiquid market.

The community found Boss Info's liquidity concerning at 91.2% free float. DQTS would leave 15% of total shares as on-chain float, in a forced auction for a nanny payroll company with low annual recurring profits. The bid side for such an auction would be essentially non-existent.

**A baseline the community has already set**

It's worth recalling that the community expressed genuine reservations about the Boss Info (BOSS) position — specifically around liquidation risk (low token float, hard to sell in a forced auction) and governance risk (bad management decisions eroding share value). Those concerns were raised against a company with CHF 14.5M parent equity, CHF 36M group revenue, CHF 5.7M operating cash flow, audited consolidated accounts, and a usable mint equal to just 29% of its equity base.

If the community found BOSS uncomfortable at those fundamentals, DQTS represents a step-change in the wrong direction on every single metric that was flagged:

- Liquidation is harder: only 56.7% of shares are even tokenized; a full position locks 73.5% of those, leaving 15% of total shares as on-chain free float vs. 91.2% for BOSS
- Governance risk is higher: the parent has 0.4% EBIT margin, CHF 8k recurring annual profit, and loss-making subsidiaries absorbing CHF 3.1M in capital
- Transparency is lower: no consolidated accounts, UK accounts unaudited, Germany undisclosed
- Equity cushion is absent: usable mint exceeds parent equity by 83%

Boss Info was a borderline case for a segment of the community. ServiceHunter is not in the same category.

**Summary**

The UK accounts add a new data point that worsens rather than improves the picture: the first subsidiary acquired under the international expansion strategy lost ~£40k in its first full year, liquidated its main tangible asset to stay solvent, and operates with declining headcount. The German subsidiary remains a black box despite operating since 2022 and absorbing CHF 700k+ in impaired loans.

The appropriate ask before this proposal moves further is straightforward: **consolidated financial statements for the group (CH + DE + UK)**. Without them, the community is being asked to underwrite risks it cannot currently quantify — at terms that the community found uncomfortable even when applied to a company with 10× the equity base and a fully disclosed operating track record.

If the community is inclined to approve anything at all, LENDS and BOSS already point in the right direction: start small, earn trust, expand later. A minting limit **below or around CHF 1M** would at least bring DQTS into the same ballpark as LENDS — a company with 3× the market cap that was approved for exactly that. CHF 5M for a business at this stage of transparency and profitability is not a calibrated risk — it is a leap of faith the protocol's collateral mechanics are not designed to absorb.

**Reply by luziusmeisser** [reply]
I'll try to keep my reply short.

1. Regarding the valuation: I don't share the provided value assessments. Paolo from Frankencoin Association came up with a fair value of 5-8 CHF per share after having studied the [consolidated accounting](https://github.com/user-attachments/files/28602679/20260528.ServiceHunter.AG.-.Review.Konzernrechnung.2025-sig-sig.pdf). Therefore, I consider the suggested liquidation price of 6 CHF defensible.
2. Regarding the substance: you are right, that substance is a critical factor, especially on a consolidated basis. The background is that the company spent its reserves on two acquisitions (earny in Switzerland, PAYE in the UK) and the German expansion. This came with heavy IT investments. But this encapsulates also some good news: the company can become profitable and build up reserves easily by refraining from further acquisitions and expansions. The Swiss core business is healthy.
3. Regarding the tokenized shares: the non-tokenized shares can be tokenized at any time on demand. They are the same share class as the already tokenized ones. In fact, I expect most of the shareholders to tokenize their shares in the coming weeks as quitt plans to advsertise the market.

I think this is a great opportunity for the Frankencoin system. But you make some valid points regarding the risks. How about reducing the proposed duration to 12 months such that we get a chance to reassess much sooner and pull the plug if the market is neglected or if there are other problems?

**Reply by samclassix** [reply]
Thank you for the consolidated accounting. A missing part for everyone who wants to study those documents and make their own pictures and valuations.

### Lets have have some math with the suggested liquidation price of 6 zchf. 

Market: 8 zchf
Liq. Price: 6 zchf
Reserve: 40%

--> threat for the equity pool starts below market prices of 3.6 zchf, which would mean a drop of -55% from the current market price. This would reflect the risk for the FPS holder. 

### 12/24 months loan duration

In my opinion, if the company is the direct or indirect beneficiary of the loan, they could still use the previous suggested 24 months loan duration. Which gives the company a bit air on the liquidity side for fixed 24 months. The "auction duration to 72 hours" is a great point worth mentioning.

### Minting Limit - start small, earn trust, expand later

In my opinion, the only thing i would adjust is to lower the minting limit and have a two step approach:

1. Proposal with 1-3M
- usable around 0.6-1.8M
- for 24 months

**grace period** for more tokenized shares so the free float ratio betters the condition, reflecting potentially smoother operations on the protocol side. 

2. After 12 months
- expect 100% mcap on-chain
- Proposal for additional 2-3M (if risks allow and/or demand increased)

**Comment by PaoloDiStef** [comment]
**Association View:**

The Association considers the proposed parameters reasonable. A company valuation of CHF 16m (CHF 8.00/share) is justified based on the consolidated financial statements, while the liquidation price of CHF 6.00 and a 40% reserve ratio serve as the primary safeguards for the system.

The global minting limit is high relative to the current size of the Frankencoin system. This can be justified during the current growth phase, but should be reassessed after the proposed 2-year maturity.

**Reasoning:**

DQTS represents exposure to a private company, so it is different from (liquid) crypto collateral. The main risk considerations are that a fair market price is more difficult to evaluate (as the company is not publicly listed), and that the possible buyer universe in a liquidation scenario is narrower because there are no clear arbitrage opportunities.

**Company Valuation**

The 2025 financial statements show that the Swiss core business is solid and profitable (CHF 5.5m revenue in 2025), while the German and UK entities are growing but currently still loss-making. Applying a 10× multiple to the Swiss entity’s CHF 1.6m EBITDA supports the proposed valuation. More sophisticated valuation methods produce the following results:

**1. Method: Adjusted net asset value**  

- Question: What is the net / adjusted value of the assets on the balance sheet?  
- Relevance: Low for SaaS companies, as IP and the customer base are not captured on the balance sheet.
- Valuation: CHF 0.2m–4.0m  

**2. Method: Earnings value**  

- Question: What is the current value of the sustainable profits, excluding growth?  
- Relevance: High; reflects a cash-flow view, but does not capture growth.
- Valuation: CHF 8.6m–16.6m 

**3. Method: Multiples valuation**

- Question: What do buyers pay for comparable companies?  
- Relevance: High; as it reflects observed market transactions.
- Valuation: CHF 19.8m–30.7m

**4. Method: Discounted cash flow (DCF)**  

- Question: What is the present value of all future free cash flows under different growth assumptions?  
- Relevance: Medium; Captures growth potential, but not well-suited from a collateral perspective as growth is uncertain.
- Valuation: CHF 26m–37m  
 
-> The earnings method and multiples-based valuation are most relevant, and both support the CHF 16m valuation.

**Reserve Requirement**

A 40% reserve requirement seems appropriate. This implies that a loss for the system would only arise if the auction were to clear below a price of CHF 3.60, corresponding to a valuation of CHF 7.2 million.

Based on the company’s business model, such a sharp decline within just 24 months seems unlikely. The tail risk is rather that there are not enough buyers during an auction, leading to a substantial discount. This is more difficult to quantify as there is no precedent for such a case. However, given the underlying business model and the likely chance that the shares could still be auctioned off at a meaningful residual value, the proposed 3% effective interest rate appears to be a reasonable compensation for FPS holders to assume this tail risk.

**Global Minting Limit**

The global minting limit relative to the company's size is significantly higher than for other tokenised share collateral on Frankencoin. @samclassix rightly flagged this as an important consideration, while @luziusmeisser 's response that non-tokenized shares can be tokenized at any time on demand helps ensure there is sufficient free float for borrowers to be challenged.

However, if the full limit is minted, a meaningful share of the ZCHF supply would be backed by DQTS, similar to the current situation with BOSS. This can be justified while Frankencoin is still growing, but should not be the case in a long-term equilibrium. A limited maturity is therefore useful, as it allows the parameters to be reassessed within a clear timeframe.

**Conclusion:**

The Association considers the proposal reasonable. A CHF 16m valuation (CHF 8.00/share) is justified based on the financial statements. The CHF 6.00 liquidation price and 40% reserve requirement largely mitigate the risks of valuation uncertainty and limited auction liquidity, while the effective 3.0% interest rate compensates FPS holders for the remaining risk.

The global minting limit is high relative to the current size of the Frankencoin system. This can be justified during the current growth phase, but should be reassessed after the proposed two-year maturity, such that the global minting limit can be resized in better proportion to the overall size of the Frankencoin system at that time.
